src/components/Pagination.test.tsx

```diff
@@ -1,110 +1,105 @@
 import { noop } from "src/utils";
 import { click, render } from "src/utils/rtl";
-import { pageOptions, Pagination, toFirstAndOffset } from "./Pagination";
+import { pageOptions, PageSettings, Pagination, toFirstAndOffset } from "./Pagination";
 
-const initPageSettings = { page: 1, perPage: 100 };
+const init: PageSettings = { pageNumber: 1, pageSize: 100 };
 
 describe("Pagination", () => {
   it("can have a data-testid", async () => {
-    const r = await render(
-      <Pagination totalCount={10} label="" setSettings={noop} settings={initPageSettings} data-testid="pag" />,
```

Comment:
I combined the two `settings= setSettings=` into a single `page=` prop that accepts the tuple of the two. This allows callers to just do:

```
const page = useState(defaultPage);

              <Pagination
                page={page}
                totalCount={totalCount}
              />
```